.github/workflows/test.yml

```diff
@@ -7,25 +7,25 @@ on:
 
   pull_request:
     paths:
-      - 'Sources/**'
-      - 'Tests/**'
-      - 'test-server/**'
-      - 'Samples/**'
-      - '.github/workflows/test.yml'
-      - 'fastlane/**'
-      - 'scripts/tests-with-thread-sanitizer.sh'
-      - 'scripts/ci-select-xcode.sh'
-      - 'scripts/xcode-test.sh'
-      - '.codecov.yml'
+      - "Sources/**"
```

Comment:
Sorry about the quotes - it's auto-formatted by vscode. I can roll this back manually if needed.